### PR TITLE
Streamification is coming

### PR DESCRIPTION
**Pull request by AWinterman**
Streamified the things! Also changed the docs slightly. The build is currently failing because of a strange npm error in node 0.8. I'm not sure what to do about that -- all tests are passing...

I'm working on an example, hopefully going to hook it up to some d3 graphs at some point, and let the user click around and make some events. https://github.com/AWinterman/estate-example

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "0.11"
+  - "0.10"
+  - "0.8"
```
Comment (AWinterman) [inline .travis.yml:5]: 0.8 and 0.6 are both failing due to npm freaking out, but I still want to test against these node versions...

**Comment by AWinterman** [comment]
hi @chrisdickinson ! I'm just closing this because 🤷

**Comment by chrisdickinson** [comment]
Oh hi! Sorry I dropped the ball on this one! Hope you're doing well!

**Comment by AWinterman** [comment]
I donno, I think this was more of a ball chucked in an arbitrary direction, so no worries!

**Pull request closed without merging**